spec.md

```diff
@@ -629,7 +642,21 @@ schema being used.
       "maintenance_info": {
         "version": "2.1.1+abcdef",
         "description": "OS image update.\nExpect downtime."
-      }
+      },
+      "extensions": [
+        {
+          "id": "urn:osbext:/v1/backup-and-restore",
+          "path": "instance-backup",
```

Comment:
This `path` does not match the format of the example (/broker-extension-path) in that it is missing a leading slash.  I think it may be worth calling out explicitly that it must start with a slash if it is going to follow the [Path Object spec](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#pathsObject)

Reply:
I think it doesn't actually need a `/`, but we should be consistent.

Reply:
I have removed the slash for consistency sake.